js/scripts.js

```diff
@@ -0,0 +1,15 @@
+var text = "Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach.";
+var animal = "Zielone słonie";
+var animalUpperCased = animal.toUpperCase();
+
+console.log(animalUpperCased);
+
+var firstWordfromText = text.slice(0,6);
```

Comment:
takie liczby jak ta `6` są słabe bo nie wiadomo skąd się wzięły. Skoro już postanowiłeś odnaleźć pierwszy wyraz to użyj https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split do podzielenia tekstu na tablicę z pojedynczymi wyrazami. Znajdź pierwszy wyraz, zmierz jego długość i zapisz to w zmiennej. Tą zmienną wtedy możesz tu użyć. Dzięki temu nie ważne jaki będzie pierwszy wyraz w zdaniu, czy długi, czy krótki - zawsze podmieniony zostanie cały wyraz.